### PR TITLE
module: lint v1

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

This is a quick pass of linting. There is still quite a bit, but I will break it up across multiple prs

### CODE REVIEW

File: module/x/gravity/keeper/genesis.go [hunk 1]
```diff
@@ -38,16 +39,20 @@ func InitGenesis(ctx sdk.Context, k Keeper, data types.GenesisState) {
...
 	for _, tx := range data.UnbatchedTransfers {
-		k.setPoolEntry(ctx, tx)
+		if err := k.setPoolEntry(ctx, tx); err != nil {
+			panic(err)
+		}
 	}
 
 	// reset attestations in state
 	for _, att := range data.Attestations {
+		att := att
```
Comment (jkilpatr) [inline module/x/gravity/keeper/genesis.go:55]: What lint does this re-declaration come from? I'm curious about the motivation. Is it to unalias a pointer?

**Review by jkilpatr — APPROVED** [review]
LGTM, a minor question but nothing worth keeping this from going in.

File: module/x/gravity/keeper/genesis.go (hunk 1 above)
Comment (marbar3778) [reply module/x/gravity/keeper/genesis.go:55]: it comes from here https://github.com/kyoh86/scopelint. It checks for unpinned variables, its more of a nit as i haven't seen it cause issues.

**Comment by marbar3778** [comment]
@jkilpatr do you know the build is and why its required in ci?

**Comment by jkilpatr** [comment]
> @jkilpatr do you know what build is and why its required in ci?

I think it's the rust build, the issue is that it's not named descriptively. Seems like a github actions issue I'll just restart it.

**Comment by jkilpatr** [comment]
well that didn't work out. I think the only way to have Github actions reset the list of tasks that need to be run is maybe for @marbar3778  to push some new code to this branch and trigger it to start again.

I don't think this has anything to do with the actual code contents of the PR. Super strange.

**Comment by marbar3778** [comment]
> > @jkilpatr do you know what build is and why its required in ci?
> 
> I think it's the rust build, the issue is that it's not named descriptively. Seems like a github actions issue I'll just restart it.

the rust build wasn't required a bit ago, did you change this?  Its not running because I told github actions to not run the rust tests if no rust is touched. We need to coordinate changes like this

**Review by fedekunze — APPROVED** [review]
ACK. Let's please get this merged and not block it on the Rust build

**Comment by jkilpatr** [comment]
> > I don't think this has anything to do with the actual code contents of the PR. Super strange. Lets try and get this unstuck and merged.
> 
> the rust build wasn't required a bit ago, did you change this? Its not running because I told github actions to not run the rust tests if no rust is touched. We need to coordinate changes like this

I did not make any modifications to the CI settings. This is an actions bug.

**Comment by jkilpatr** [comment]
@marbar3778 If you could close this pr and open a new one (with identical contents) I think that will resolve this issue. At least hopefully.

**Comment by marbar3778** [comment]
> @marbar3778 If you could close this pr and open a new one (with identical contents) I think that will resolve this issue. At least hopefully.

fixed it. I unrequired the rust build since we dont run it eeach time.

**Pull request merged**